packages/palette/src/elements/CleanTag/CleanTag.tsx

```diff
@@ -9,28 +9,51 @@
  * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
  */
 import React, { ComponentClass, FunctionComponent } from "react"
-import { styles, StylesProps } from "styled-system"
 import { View } from "../../platform/primitives"
 
-const allPropTypes: Partial<StylesProps> = Object.keys(styles)
-  .filter(key => typeof styles[key] === "function")
-  .reduce(
-    (styleProps: Partial<StylesProps>, key) => ({
-      ...styleProps,
-      ...styles[key].propTypes,
-    }),
-    {}
-  )
-
 /**
  * The default set of props to remove from components rendered by styled-components
  */
-export const omitProps = [...Object.keys(allPropTypes), "theme"]
+export const omitProps = [
+  "m",
+  "mx",
+  "my",
+  "ml",
+  "mr",
+  "mt",
+  "mb",
+  "p",
+  "pt",
+  "pb",
+  "pl",
+  "pr",
+  "px",
+  "py",
+  "width",
+  "height",
+  "maxWidth",
+  "maxHeight",
+  "minWidth",
+  "minHeight",
+  "fontSize",
+  "fontWeight",
+  "fontFamily",
+  "fontStyle",
+  "color",
+  "backgroundColor",
+  "bg",
+  "textAlign",
+  "lineHeight",
+  "letterSpacing",
+  "display",
+  "theme",
+  "borderRadius",
+]
```

Comment:
Can we alpha this? There's some grouping here which makes sense, but won't make sense for the next dev quickly trying to see if something is in the list.